app/RowItem.js

```diff
@@ -42,6 +43,9 @@ class RowItem extends React.Component {
             position: "relative",
             width: "100%",
         };
+
+        var sequence = (uppercase) ? row.sequence.toUpperCase() : row.sequence.toLowerCase();
```

Comment:
@jakecoble it seems like we might want to have 3 options here, Upper, Lower, and "mixed"? What do you think?